json_serialization/reader.nim

```diff
@@ -176,7 +176,11 @@ proc parseJsonNode(r: var JsonReader): JsonNode =
     r.lexer.next()
     if r.lexer.tok != tkCurlyRi:
       while r.lexer.tok == tkString:
-        r.readJsonNodeField(result.fields.mgetOrPut(r.lexer.strVal, nil))
+        r.readJsonNodeField(
+          try:
+            result.fields.mgetOrPut(r.lexer.strVal, nil)
+          except KeyError:
```

Comment:
it doesn't.. this shouldn't be needed

Reply:
The compiler required this. I need to investigate why.